packages/gitui-git/gitui-git.pacscript

```diff
@@ -0,0 +1,37 @@
+pkgname="gitui-git"
+gives="gitui"
+pkgdesc="Blazing fast terminal-ui for git written in Rust"
+url="https://github.com/extrawurst/gitui"
+source=("https://github.com/extrawurst/gitui.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="0.26.3"
+depends=('gcc' 'libgit2-dev' 'libxcb1' 'zlib1g-dev')
+makedepends=('git' 'cargo' 'python3') # xcb crate needs python
+external_connection="true"
+
+prepare() {
+  cd "${_archive}"
```

Comment:
Do not use _archive, use something like `"${gives}"` or `"gitui"`.

Reply:
Alright, will do